planning.py

```diff
@@ -294,6 +329,126 @@ def add_vertices(
                     if self.workspace.edge_is_in_collision(vo.coord, v.coord):
                         continue
                     vo.connect(v)
+        self.preprocessing_time = time.time()-start_time
+
+    def RRT(self,
```

Comment:
I would make the `RRT` as well as the double trees approach below separate classes from the `RRG`.

Reply:
I'll get on that!